src/service/resources-impl.js

```diff
@@ -859,6 +871,19 @@ export class Resources {
     this.visibilityStateMachine_.setState(
       this.viewer_.getVisibilityState()
     );
+
+    if (this.maybeChangeHeight_) {
+      this.maybeChangeHeight_ = false;
+      this.vsync_.measure(() => {
```

Comment:
Actually, I think `doPass_` happens less frequently than `vsync.measure`, maybe there's no need to dedupe?